R/geom-ribbon.r

```diff
@@ -2,7 +2,8 @@
 #'
 #' For each x value, `geom_ribbon` displays a y interval defined
 #' by `ymin` and `ymax`. `geom_area` is a special case of
-#' `geom_ribbon`, where the `ymin` is fixed to 0.
+#' `geom_ribbon`, where the `ymin` is fixed to 0 and `y` is used instead
```

Comment:
Since you're in here, can you please add `()` after function names?

Reply:
sure!  added () to the function names